inventory/group_vars/all.yml

```diff
@@ -67,6 +67,20 @@ env:
   RAILS_ENV: "{{ rails_env }}"
   PATH: "{{ gem_home }}/bin:{{ ansible_env.PATH }}"
 
+multiredis_instances:
+  - name: cache
+    port: 6380
+    configs: |
+      maxmemory 124mb
+      maxmemory-policy allkeys-lru
+  - name: jobs
+    port: 6381
+    configs: |
+      maxmemory 50mb
```

Comment:
Actual memory usage of this second Redis instance (from the Sidekiq dashboard):

![Screenshot from 2021-05-31 12-08-32](https://user-images.githubusercontent.com/9029026/120184625-7cdca680-c211-11eb-8890-c6a10b9c776f.png)